doc/Learn/Tutorials/Markup/HowTo-CustomMarkupProject-Theme.md

```diff
@@ -0,0 +1,312 @@
+
+It contains two libraries
+
+- [Material Overview](https://platform.uno/docs/articles/external/uno.themes/doc/material-getting-started.html)
+
+- [Cupertino Overview](https://platform.uno/docs/articles/external/uno.themes/doc/cupertino-getting-started.html)
+
+To help you style your application with a few lines of code to include and change:
+
+- Color system for both Light and Dark theme
+- Styles for existing WinUI controls like Buttons, TextBox, etc.
+- Change the Font of the Application
+
+## Quick Start
+
+In the two links above, you have a Getting Started for Material and Cupertino. 
+
+But the difference between they are the references that can be included on the Solution.
+
+You may notice that there are only minor differences in the XAML configuration in the link above and in this current tutorial.
+
+Since we are using the Default Fluent Theme in the current project, we can choose to stay with it or add a different one.
+
+Below we can see how to include and how to customize each of the two.
+
+##  Material
+
+Assuming that we are on the Fluent Theme and we want to change to the Material Theme, we just need to change some files (Or add some Nuget References) and it is done.
+
+### Below is the list of files we need to modify
+
+#### In the File Directory.Packages.props
+
+In the `Directory.Packages.props`, we need to add the `PackageReference` below or using the Manage NuGet Packages Manager and update the versions for the project.
+
+```xml
+    <PackageVersion Include="Uno.Material.WinUI.Markup" Version="0.0.0" />	
+    <PackageVersion Include="Uno.Material.WinUI" Version="0.0.0" />	
+    <PackageVersion Include="Uno.Dsp.Tasks" Version="0.0.0" />
+```
+--------------------------------------------------------------
+If you prefer you can use the Manage NuGet Packages:
+In the Solution Explorer panel, right-click on your app's App Code Library project (PROJECT_NAME.csproj)
+and select Manage NuGet Packages...
+
+**Install the following Packages**
+
+- `Material.WinUI.Markup`, 
+- `Uno.Material.WinUI` 
+- `Uno.Dsp.Tasks`
+--------------------------------------------------------------
+
+#### In the File of Projects References
+
+Add PackageReference to the PROJECT_NAME.csproj and the PROJECT_NAME.[Platform].csproj projects.
+
+```xml
+<PackageReference Include="Uno.Material.WinUI.Markup" />	
+<PackageReference Include="Uno.Material.WinUI" />
+<PackageReference Include="Uno.Dsp.Tasks" />
+```
+
+#### In the File GlobalUsings.cs
+```csharp
+global using Uno.Material;
+```
+
+**And it is that, we now have the Material Style in your application.**
+
+#### In the File AppResources.cs
+
+But in case we want some customization, we need to override the Colors and the Fonts.
+
+First we need to create a folder named `Styles` in the shared class library. Then Create a new class file to override the ColorPalette, we named `ColorPaletteOverride.cs` and add the content below.
```

Comment:
```suggestion
#### In the File GlobalUsings.cs
```csharp
global using Uno.Material;
```

**And it is that we now have the Material Style in your application.**

#### In the File AppResources.cs

But in case we want some customization, we need to override the Colors and the Fonts.

First, we need to create a folder named `Styles` in the shared class library. Then Create a new class file to override the ColorPalette, which we named `ColorPaletteOverride.cs`, and add the content below.
```